Modpack gregtech-community-pack-modern fails with InvalidFormatException

```
[mc-image-helper] 03:18:12.972 ERROR : 'install-curseforge' command failed. Version is 1.50.2
2025-10-31T03:18:12.973953417Z reactor.core.Exceptions$ReactiveException: me.itzg.helpers.http.ResponseParsingException: Failed to parse response body into class me.itzg.helpers.curseforge.model.GetModFileResponse
2025-10-31T03:18:12.973955917Z 	at reactor.core.Exceptions.propagate(Exceptions.java:410)
2025-10-31T03:18:12.973956959Z 	at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:102)
2025-10-31T03:18:12.973957917Z 	at reactor.core.publisher.Mono.block(Mono.java:1779)
2025-10-31T03:18:12.973959084Z 	at me.itzg.helpers.curseforge.CurseForgeInstaller.processModpack(CurseForgeInstaller.java:600)
2025-10-31T03:18:12.973960084Z 	at me.itzg.helpers.curseforge.CurseForgeInstaller.resolveModpackFileAndProcess(CurseForgeInstaller.java:414)
2025-10-31T03:18:12.973961042Z 	at me.itzg.helpers.curseforge.CurseForgeInstaller.installByRetrievingModpackZip(CurseForgeInstaller.java:274)
2025-10-31T03:18:12.973982792Z 	at me.itzg.helpers.curseforge.CurseForgeInstaller.lambda$install$3(CurseForgeInstaller.java:196)
2025-10-31T03:18:12.973984250Z 	at me.itzg.helpers.curseforge.CurseForgeInstaller.install(CurseForgeInstaller.java:242)
2025-10-31T03:18:12.973985209Z 	at me.itzg.helpers.curseforge.CurseForgeInstaller.install(CurseForgeInstaller.java:195)
2025-10-31T03:18:12.973986084Z 	at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:252)
2025-10-31T03:18:12.973987000Z 	at me.itzg.helpers.curseforge.InstallCurseForgeCommand.call(InstallCurseForgeCommand.java:33)
2025-10-31T03:18:12.973987959Z 	at picocli.CommandLine.executeUserObject(CommandLine.java:2031)
2025-10-31T03:18:12.973988834Z 	at picocli.CommandLine.access$1500(CommandLine.java:148)
2025-10-31T03:18:12.973989709Z 	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2469)
2025-10-31T03:18:12.973990625Z 	at picocli.CommandLine$RunLast.handle(CommandLine.java:2461)
2025-10-31T03:18:12.973991459Z 	at picocli.CommandLine$RunLast.handle(CommandLine.java:2423)
2025-10-31T03:18:12.973992334Z 	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
2025-10-31T03:18:12.973993209Z 	at picocli.CommandLine$RunLast.execute(CommandLine.java:2425)
2025-10-31T03:18:12.973994042Z 	at picocli.CommandLine.execute(CommandLine.java:2174)
2025-10-31T03:18:12.973994875Z 	at me.itzg.helpers.McImageHelper.main(McImageHelper.java:176)
2025-10-31T03:18:12.973995709Z 	Suppressed: java.lang.Exception: #block terminated with an error
2025-10-31T03:18:12.973997500Z 		at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:104)
2025-10-31T03:18:12.973998500Z 		... 18 common frames omitted
2025-10-31T03:18:12.973999375Z Caused by: me.itzg.helpers.http.ResponseParsingException: Failed to parse response body into class me.itzg.helpers.curseforge.model.GetModFileResponse
2025-10-31T03:18:12.974000625Z 	at me.itzg.helpers.http.ObjectFetchBuilder.lambda$handleResponse$1(ObjectFetchBuilder.java:87)
2025-10-31T03:18:12.974001792Z 	Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
2025-10-31T03:18:12.974002667Z Assembly trace from producer [reactor.core.publisher.MonoMap] :
2025-10-31T03:18:12.974003584Z 	reactor.core.publisher.Mono.checkpoint(Mono.java:2261)
2025-10-31T03:18:12.974004459Z 	me.itzg.helpers.curseforge.CurseForgeApiClient.getModFileInfo(CurseForgeApiClient.java:248)
2025-10-31T03:18:12.974005334Z Error has been observed at the following site(s):
2025-10-31T03:18:12.974006209Z 	*__checkpoint() ⇢ at me.itzg.helpers.curseforge.CurseForgeApiClient.getModFileInfo(CurseForgeApiClient.java:248)
2025-10-31T03:18:12.974007584Z 	*____checkpoint ⇢ Processing file  1310132:7036004 from modpack
2025-10-31T03:18:12.974009625Z 	|_ checkpoint() ⇢ at me.itzg.helpers.curseforge.CurseForgeInstaller.lambda$processModpack$15(CurseForgeInstaller.java:596)
2025-10-31T03:18:12.974010709Z Original Stack Trace:
2025-10-31T03:18:12.974011500Z 		at me.itzg.helpers.http.ObjectFetchBuilder.lambda$handleResponse$1(ObjectFetchBuilder.java:87)
2025-10-31T03:18:12.974012875Z 		at reactor.core.publisher.MonoFlatMap$FlatMapMain.onNext(MonoFlatMap.java:132)
2025-10-31T03:18:12.974016334Z 		at reactor.core.publisher.MonoPublishOn$PublishOnSubscriber.run(MonoPublishOn.java:181)
2025-10-31T03:18:12.974017250Z 		at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:68)
2025-10-31T03:18:12.974018125Z 		at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:28)
2025-10-31T03:18:12.974019000Z 		at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
2025-10-31T03:18:12.974019875Z 		at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
2025-10-31T03:18:12.974020834Z 		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
2025-10-31T03:18:12.974021709Z 		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
2025-10-31T03:18:12.974022625Z 		at java.base/java.lang.Thread.run(Unknown Source)
2025-10-31T03:18:12.974024000Z Caused by: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `me.itzg.helpers.curseforge.model.FileStatus` from String "20": not one of the values accepted for Enum class: [11, 12, 13, 14, 15, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
2025-10-31T03:18:12.974025750Z  at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 196] (through reference chain: me.itzg.helpers.curseforge.model.GetModFileResponse["data"]->me.itzg.helpers.curseforge.model.CurseForgeFile["fileStatus"])
2025-10-31T03:18:12.974027209Z 	at com.fasterxml.jackson.databind.exc.InvalidFormatException.from(InvalidFormatException.java:67)
2025-10-31T03:18:12.974028500Z 	at com.fasterxml.jackson.databind.DeserializationContext.weirdStringException(DeserializationContext.java:1986)
2025-10-31T03:18:12.974029459Z 	at com.fasterxml.jackson.databind.DeserializationContext.handleWeirdStringValue(DeserializationContext.java:1272)
2025-10-31T03:18:12.974030459Z 	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer._deserializeAltString(EnumDeserializer.java:446)
2025-10-31T03:18:12.974031375Z 	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer._fromString(EnumDeserializer.java:303)
2025-10-31T03:18:12.974032292Z 	at com.fasterxml.jackson.databind.deser.std.EnumDeserializer.deserialize(EnumDeserializer.java:282)
2025-10-31T03:18:12.974033209Z 	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
2025-10-31T03:18:12.974034125Z 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:302)
2025-10-31T03:18:12.974035042Z 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:169)
2025-10-31T03:18:12.974038000Z 	at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
2025-10-31T03:18:12.974038875Z 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:302)
2025-10-31T03:18:12.974039792Z 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:169)
2025-10-31T03:18:12.974040917Z 	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342)
2025-10-31T03:18:12.974044834Z 	at com.fasterxml.jackson.databind.ObjectReader._bindAndClose(ObjectReader.java:2130)
2025-10-31T03:18:12.974045709Z 	at com.fasterxml.jackson.databind.ObjectReader.readValue(ObjectReader.java:1500)
2025-10-31T03:18:12.974046584Z 	at me.itzg.helpers.http.ObjectFetchBuilder.lambda$handleResponse$1(ObjectFetchBuilder.java:83)
2025-10-31T03:18:12.974047500Z 	at reactor.core.publisher.MonoFlatMap$FlatMapMain.onNext(MonoFlatMap.java:132)
2025-10-31T03:18:12.974048375Z 	at reactor.core.publisher.MonoPublishOn$PublishOnSubscriber.run(MonoPublishOn.java:181)
2025-10-31T03:18:12.974049792Z 	at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:68)
2025-10-31T03:18:12.974050709Z 	at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:28)
2025-10-31T03:18:12.974051542Z 	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
2025-10-31T03:18:12.974052500Z 	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
2025-10-31T03:18:12.974053542Z 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
2025-10-31T03:18:12.974054417Z 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
2025-10-31T03:18:12.974055292Z 	at java.base/java.lang.Thread.run(Unknown Source)
2025-10-31T03:18:13.367056459Z [init] [ERROR] Failed to auto-install CurseForge modpack
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -49,7 +49,7 @@ RUN easy-add --var os=${TARGETOS} --var arch=${TARGETARCH}${TARGETVARIANT} \
   --var version=${MC_SERVER_RUNNER_VERSION} --var app=mc-server-runner --file {{.app}} \
   --from ${GITHUB_BASEURL}/itzg/{{.app}}/releases/download/{{.version}}/{{.app}}_{{.version}}_{{.os}}_{{.arch}}.tar.gz
 
-ARG MC_HELPER_VERSION=1.50.2
+ARG MC_HELPER_VERSION=1.50.3
 ARG MC_HELPER_BASE_URL=${GITHUB_BASEURL}/itzg/mc-image-helper/releases/download/${MC_HELPER_VERSION}
 # used for cache busting local copy of mc-image-helper
 ARG MC_HELPER_REV=1

